src/pwa-buildpack/project-setup/create-configuration-files/index.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Create configuration files
+---
+
+In the [previous topic], you created a `package.json` file and installed the project dependencies.
+In this topic, you will create the other configuration files for your development environment.
+
+## Create the Babel configuration file
+Create a `.babelrc` file in your theme’s root directory with the following content:
+
+``` javascript
+{
+    "plugins": [
+        "syntax-jsx",
+        "transform-class-properties",
+        "transform-object-rest-spread",
+        "transform-react-jsx"
+    ]
+}
+```
+
+This Babel configuration file allows you to use [JSX] in your project.
```

Comment:
This is a little misleading; Babel is for any and all JS code transpilation and modification, not just JSX. In this `.babelrc` we tell Babel to transform JSX, and also to transpile a couple other features into backwards-compatible JavaScript. Like JSX, these are for convenience and cleaner-looking code. Our plan is to include the features we want to use one by one.

The `transform-class-properties` plugin will turn this more modern code:

```js
class Example {
  anInstanceProperty = 52;
}
```

into this more backwards-compatible code:

```js
class Example {
  constructor() {
    this.anInstanceProperty = 52;
  }
}
```

And `transform-object-rest-spread` turns this modern code:

```js
const config = { ...defaultConfig, ...passedConfig };
```

Into this:

```js
const config = Object.assign({}, defaultConfig, passedConfig);
```

We don't need to tell them all that, but there's your context; Babel is for general operations like this.